### DISCUSSION TITLE
Question about GitHub Pages

### DISCUSSION THREAD

**Discussion by mate0444** [community/community#120739] [Other Feature Feedback, Questions, & Ideas]
Hello, I had a query of github pages since it happened to me that I uploaded my project to pages but my mom does not have a github account and she does not see the option "deployments" as to get into my page, and she does not use the code since she does not understand much about it. The same thing happens when I open the github on the web without an account I can't see the link of the project what I uploaded, how is that? you have to do something special?

**Comment by eddysantiagoo** [comment]
Hola @mate0444 como vas, creo que el problema que tienes esta referido a que no puedes ver tu proyecto desplegado con github pages, cuando despliegas un sitio con github pages automáticamente github crea una ruta como: https://mate0444.github.io/"el-nombre-de-tu-repo" ejemplo: https://mate0444.github.io/PROYECTO-FINAL-PORTFOLIO/ 

Con compartir el link no es necesario que la otra persona tenga cuenta de GitHub ya que actualmente ya lo tienes público en la web

Si esta respuesta te ayudo marca esta como la repuesta correcta por favor

**Comment by mate0444** [comment]
si, pero ahi esta el problema, yo quiero poner mi github en el CV pero si yo pongo el link de mi perfil de GitHub la gente va a ver mis proyectos con el codigo, pero no como mostraria la pagina, se entiende?

**Reply by Mazp17** [reply]
Según entiendo tu quieres que se muestre la pagina, en este caso tu [portofolio](https://mate0444.github.io/PROYECTO-FINAL-PORTFOLIO/).
Pero cuando pones el GitHub te muestra el código y no hay forma de mirar el proyecto?
Si es este el caso te recomiendo que pongas el link directamente. Puedes hacer un apartado para proyectos y poner dos links uno que lleve a la pagina en si y otro al repositorio. 
O en su defecto, en el about del proyecto puedes poner el link del proyecto en el input website.

**Reply by mate0444** [reply]
no, mi problema es que yo desde mi perfil y con mi cuenta puedo ver bien mi pagina y me sale todo, la pagina el codigo, todo, pero al momento en que mi familia entra sin cuenta a github ya que no estan vinculados con la programacion, abren el repositorio pero no les aparece el deploy para abrir la pagina que creo ese codigo
![image](https://github.com/community/community/assets/132771918/eae0fc03-9932-4d6c-84cd-b03518301699)
![image](https://github.com/community/community/assets/132771918/a163ae70-1608-4689-b9ae-70ee4178a56e)

**Reply by eddysantiagoo** [reply]
@mate0444 Vale, lo más fácil si lo que quieres es que una persona al ingresar vea el link de tu web sin saber de codigo o sin registrarse es agregando la URL de tu proyecto en la sección "about" con esto podrías mostrar la URL de tu web sin necesidad de que un usuario esté registrado en github
![Screenshot_20240422_152733_Edge](https://github.com/community/community/assets/85720891/81e9fa97-b9d7-4fbb-ab45-30d901929f83)
